Runtimes/Supplemental/cmake/modules/FindSwiftCore.cmake

```diff
@@ -114,6 +114,48 @@ elseif(WIN32)
     INTERFACE_INCLUDE_DIRECTORIES "${SwiftCore_INCLUDE_DIR}")
   find_package_handle_standard_args(SwiftCore DEFAULT_MSG
     SwiftCore_LIBRARY SwiftCore_INCLUDE_DIR)
+elseif(ANDROID)
+  if(BUILD_SHARED_LIBS)
+    find_path(SwiftCore_INCLUDE_DIR
+      "Swift.swiftmodule"
+      NO_CMAKE_FIND_ROOT_PATH
```

Comment:
To double check my understanding: are you using `NO_CMAKE_FIND_ROOT_PATH` to signal that we are not mixing and matching target and host environments?

Reply:
No, this is required to ensure that we do not mix and match the Android sysroot and the Swift sysroot. When building for Android, we would search for the path "[Android sysroot][Swift sysroot]" which would not exist as the paths are concatenated.